tests/integration/distributed/test_index_query/test_integration.py

```diff
@@ -36,7 +36,7 @@ def test_index_query():
         payload={'top_k': 10, 'data': ['text:hey, dude']},
     )['index']['docs'][0]['text']
 
-    print(f'Indexed document has the text: {text_indexed}')
+    assert text_indexed == 'text:hey, dude'
```

Comment:
can we do this for the rest of the tests? do more asserts in the middle of the test?

Reply:
I added them to the other ones also